src/main/webapp/js/playGame.js

```diff
@@ -0,0 +1,146 @@
+async function initMapPlayGame() {
+  const params = new URLSearchParams();
+  params.append('gameID', 'demogameid')
+  var request = new Request('/load-game-data', {method: 'POST', body: params});
+  fetch(request).then(response => response.json()).then(async (data) => {
+    var stage1 =  await getStage(data.stages[0]);
+    var startingLocation = {lat: stage1.startingLocation.latitude, lng: stage1.startingLocation.longitude};
+    var map = new google.maps.Map(
+      document.getElementById('playMap'), {
+      center: startingLocation, 
+      gestureHandling: 'greedy',
+      streetViewControl: false
+    });
+
+    getHints(stage1).forEach(hint => 
+      addHintMarker(map, {lat: hint.location.latitude, lng: hint.location.longitude}, hint.text, hint.hintNumber)
+    );
+
+    var panorama = map.getStreetView();
+    var panoramaOptions = {
+      enableCloseButton:false
+    };
+    panorama.setOptions(panoramaOptions);
+    panorama.setPosition(startingLocation);
+    panorama.setVisible(true);
+
+    createGameInfoOnSideOfMap(data, stage1);
+  });
+}
+
+function createGameInfoOnSideOfMap(data, stage) {
+    var gameInfo = document.getElementById('game-info');
+    
+    var gameName = document.createElement('h1');
+    gameName.innerHTML = data.gameName;
+    gameName.className = 'center'
+    gameInfo.appendChild(gameName);
+
+    var gameStage = document.createElement('h2');
+    gameStage.innerHTML = 'You are on stage:' + stage.stageNumber + '/' + data.stages.length;
+    gameStage.className = 'center'
+    gameInfo.appendChild(gameStage);
+
+    var theWordHints = document.createElement('h3');
+    theWordHints.innerHTML = 'Hints:';
+    gameInfo.appendChild(theWordHints);
+
+    var starterHint = document.createElement('h3');
+    starterHint.innerHTML = 'Starter: ' + stage.startingHint;
+    gameInfo.appendChild(starterHint);
+
+    var hintsDiv = document.createElement('div');
+    var hintsOl = document.createElement('ol');
+    hintsOl.id = 'hints';
+
+    getHints(stage).forEach(hint => 
+      hintsOl.appendChild(createHintPlaceHolder(hint.hintNumber))
```

Comment:
In the lines above, I have:

```
var hintsDiv = document.createElement('div');
var hintsOl = document.createElement('ol');
hintsOl.id = 'hints';
```
From here I thought this would be clear that it is a div and ol and because its called hintsDiv and hintsOl, that it's a div and ol for hints. So, these are not abbreviations but I will add comments to clarify this.